hotshot-query-service/src/data_source/storage/sql.rs

```diff
@@ -796,6 +802,114 @@ impl VersionedDataSource for SqlStorage {
     }
 }
 
+struct Leaf2Row {
+    height: i64,
+    hash: String,
+    block_hash: String,
+    leaf2: Value,
+    qc2: Value,
+}
+impl SqlStorage {
+    pub async fn migrate_types<Types: NodeType>(&self) -> anyhow::Result<()> {
+        let mut offset = 0;
+        let limit = 10000;
+
+        loop {
```

Comment:
Lets' add some logs

1. Inform about start of migration
2. Inform about progress, unless it
3. Inform about completion.